conf/evolutions/reversions/119-add-metadata-to-folders-and-datasets.sql

```diff
@@ -0,0 +1,68 @@
+START TRANSACTION;
+
+do $$ begin ASSERT (select schemaVersion from webknossos.releaseInformation) = 119, 'Previous schema version mismatch'; end; $$ LANGUAGE plpgsql;
+
+DROP VIEW webknossos.folders_;
+DROP VIEW webknossos.datasets_;
+
+-- Folder part
+ALTER TABLE webknossos.folders DROP COLUMN metadata;
+
+-- Dataset part
+ALTER TABLE webknossos.datasets ADD COLUMN details JSONB;
+ALTER TABLE webknossos.datasets ADD CONSTRAINT detailsIsJsonObject CHECK(jsonb_typeof(details) = 'object');
+
+-- Add existing info on species of metadata to details
+UPDATE webknossos.datasets
+SET details = jsonb_set(COALESCE(details, '{}'), '{species}', (
```

Comment:
The newest commit changed 
```
SET details = jsonb_set(details, '{species}', (
```
to
```
SET details = jsonb_set(COALESCE(details, '{}'), '{species}', (
```

This was necessary, as the default value of the newly re-added details column was null and thus the `jsonb_set` would also return null. This should work now :)